server/src/config/config.js

```diff
@@ -94,6 +94,51 @@ const config = convict({
         }
       }
     }
+  },
+  mailer: {
+    fake: {
+      doc: 'True to use a fake smtp',
+      format: Boolean,
+      default: true,
+      env: 'MAILER_FAKE'
+    },
+    from: {
+      doc: 'Source email address',
+      format: String,
+      default: 'Raven-Build <build@raven-os.fr>',
+      env: 'MAILER_FROM'
+    },
+    transporter: {
+      host: {
+        doc: 'Host of the smtp server',
+        format: String,
+        default: undefined,
```

Comment:
Default value are great to identify the format of the parameter

Reply:
the format is `String` and the default value is `undefined` there is no default value for smtp we use a fake smtp for develoment

Reply:
You can use `localhost`, and a format is different than a type, it describes what the string should contain.

Reply:
done